broadinstitute/pearl
# JN-31 diffing portal environments

This adds capability for computing the difference between two portal environments (note that this does not yet include study environments).  The UX is very much a work-in-progress, and will probably change dramatically over the next few weeks as we learn what's actually useful in seeing and moving stuff around.  It does add a new consolidated Portal Environment view that I think is more useful than what we had before (at least it makes it easier to get to the Participants)

Next steps will be to add select and "update" capabilities to the diff page so that changes can be moved between environments.

TO TEST:
1. restart adminApiApp, and repopulate ourhealth
2. go to https://localhost:3000/ourhealth
![image](https://user-images.githubusercontent.com/2800795/229292480-7887cc2b-2a12-4f97-ae71-3b7f4d5de5bd.png)
3. Click "copy from sandbox" in the irb environment
4. See the differences between the sandbox and irb environment
![image](https://user-images.githubusercontent.com/2800795/229292537-5aee81be-0223-4aef-aeab-ca1428a3756a.png)
